[1.32.0] Alert history (#650)
conduktor/conduktor-docs: docs/platform/navigation/settings/integrations.md

---
sidebar_position: 6
title: Integrations
description: Connect Conduktor to other tools.
---

## Configuring integrations

Conduktor Console can integrate with Slack, MS Teams and any system that receives webhooks.

Enabling the **Slack** integration requires creating a Slack application, installing this to your Slack workspace, inviting your app to the channel and adding the application token to Conduktor.  

> Enabling the **MS Teams** integration requires a webhook URL from Teams when creating alerts with this destination.

Author: @RG-conduktor thoughts?

```suggestion
After enabling the **MS Teams** integration you will need a Teams webhook URL to create alerts with this destination.
```
